### PR TITLE
Add SamplingCollection

### LINKED ISSUE #189: Missing SamplingCollection
I noticed that we have added `ObservationCollection`, `ActuationCollection` and `SampleCollection` to SOSA.

But the actual equivalent of `ObservationCollection` and `ActuationCollection` in the sampling module would be `SamplingCollection`.

While `SampleCollection` is also useful, I was wondering if the omission of `SamplingCollection` was intentional.

### PR DESCRIPTION
**Pull request by ashleysommer**
Added `SamplingCollection` following the patterns introduced by `ActuationCollection` and `ObservationCollection`.
Updated diagrams with the new collection type.
Fixes #189

### CODE REVIEW

**Comment by ashleysommer** [comment]
I'll create a PR to the TERN ontology that profiles SOSA, to provide an implementation of this feature.

**Comment by ashleysommer** [comment]
I created a W3.org account using my CSIRO email, and linked it to my GitHub account as instructed. But I don't have permission to retrigger the the IPR check.

**Comment by ashleysommer** [comment]
Added implementation in TERN Ontology https://github.com/ternaustralia/ontology_tern/pull/221

@dr-shorthair Is there anything remaining that I need to put in place for this?

**Comment by dr-shorthair** [comment]
> I created a W3.org account using my CSIRO email, and linked it to my GitHub account as instructed. But I don't have permission to retrigger the the IPR check.

@bert-github I tried to re-trigger IPR check but something is awry. 
Can you help? 

@ashleysommer is a former colleague of mine from CSIRO. 
What does he have to do to clear the IPR issue?

**Comment by dr-shorthair** [comment]
@ashleysommer could you merge the subsequent changes made to `gh-pages` into your branch, and remove conflicts?

File: ssn/rdf/ontology/core/sosa-sampling.ttl [hunk 1]
```diff
@@ -202,6 +202,30 @@ sosa:Sampling
   skos:example "Taking a diamond-drill core from a rock outcrop."@en ;
   rdfs:isDefinedBy sosa-sam: .
 
+sosa:SamplingCollection 
+  a owl:Class ;
+  rdfs:label "Sampling Collection"@en ;
+  skos:definition """
+  Collection of one or more samplings or sampling collections, i.e. collections may be nested. 
+
+  The following Sampling properties may appear in an Sampling Collection:
```
Comment (hylkevds) [inline ssn/rdf/ontology/core/sosa-sampling.ttl]: a, not an.

**Review by hylkevds — APPROVED** [review]
Looks good, just one typo

File: ssn/chapters/Sampling.html
Comment (KathiSchleidt) [inline ssn/chapters/Sampling.html:1]: "properties may appear in an <a href="#SOSASamplingCollection"
should be:
"properties may appear in a <a href="#SOSASamplingCollection"
a, not an

File: ssn/rdf/ontology/core/sosa-sampling.ttl (hunk 1 above)
Comment (KathiSchleidt) [reply ssn/rdf/ontology/core/sosa-sampling.ttl]: Agreed, recurring problem, also identified above

**Review by KathiSchleidt — COMMENTED** [review]
Except for a few grammar glitches, "an SamplingCollection" in place of  "a SamplingCollection" (also addressed by Hylke), looks good to go :)

**Comment by dr-shorthair** [comment]
I fixed the grammar error. 
The bigger job was to merge with the current head 

See https://github.com/w3c/sdw-sosa-ssn/issues/189#issuecomment-2357433910

If that branch is good then we can make a new PR to replace this PR #227 (which relates to Ashley's repo).

**Comment by ashleysommer** [comment]
@dr-shorthair I've merged the PR on the branch, this should be up to date now.

**Review by dr-shorthair — APPROVED** [review]

**Pull request merged**